backend/server/routes.js

```diff
@@ -15,6 +16,8 @@ module.exports = {
       res.send('{"message":"Developer Profiles API"}');
     });
 
+    server.use('/api/saveresethash', forgotPasswordRouter);
```

Comment:
You don't need to tell API users that we are resetting a hash. I would change it to `/api/reset-password`